docs/_layouts/no-nav-page.html

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+---
+
+{% assign current_dir = page.dir %}
+{% assign paths = page.dir | split: "/" %}
+{% assign parent_dir = paths[1] %}
+
+{%-
+assign nav_pages = site.pages
+| where_exp: "item", "item.dir contains parent_dir"
+-%}
+
+{% assign first_level_dir = parent_dir | prepend: '/' | append: '/' %}
+{%-
+assign first_level = nav_pages
+| where_exp: "item", "item.dir == first_level_dir"
+| group_by: "section"
+| sort: 'name', 'last'
+-%}
+
+<div class="nhsuk-width-container">
+  <main class="nhsuk-main-wrapper" id="maincontent" role="main">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <div class="nhsnotify-pane__main-content nhsuk-grid-column-three-quarters">
```

Comment:
You shouldn't need the `.nhsuk-grid-column-full` container because the page width is specified in the next container: `.nhsuk-grid-column-three-quarters`

Reply:
Nice, will remove